Dockerfile

```
@@ -0,0 +1,39 @@
+FROM debian:10.5-slim as base
+
+RUN apt-get update \
+  && apt-get install -y --no-install-recommends \
+     bash \
+     jq \
+     less \
+     moreutils \
+     procps \
+  && apt-get clean
```

```suggestion
  && rm -rf /var/lib/apt/lists/*
```

https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#run

> Official Debian and Ubuntu images automatically run apt-get clean, so explicit invocation is not required.